src/Post.php

```diff
@@ -0,0 +1,364 @@
+<?php
+
+namespace Lullabot\Parsely;
+
+/**
+ * Represents a Post.
+ *
+ * @todo Is this data model shared for all posts? The API is not REST so it's
+ * hard to tell until we need to load posts from another resource.
+ *
+ * @see https://www.parse.ly/help/api/analytics/
+ */
+class Post
```

Comment:
All the methods from here on are the various properties documented by Parse.ly. There's nothing more than generated getters and setters.